var/spack/repos/builtin/packages/py-pint/package.py

```diff
@@ -12,9 +12,19 @@ class PyPint(PythonPackage):
     It allows arithmetic operations between them and conversions from and
     to different units."""
 
-    homepage = "https://pypi.python.org/pypi/pint"
-    url      = "https://pypi.io/packages/source/p/pint/Pint-0.8.1.tar.gz"
+    homepage = "https://pypi.org/project/Pint"
+    url = "https://pypi.io/packages/source/p/pint/Pint-0.11.tar.gz"
 
-    version('0.8.1', sha256='afcf31443a478c32bbac4b00337ee9026a13d0e2ac83d30c79151462513bb0d4')
+    version(
+        '0.11', sha256='308f1070500e102f83b6adfca6db53debfce2ffc5d3cbe3f6c367da359b5cf4d')
+    version('0.10.1', sha256='d739c364b8326fe3d70773d5720fa8b005ea6158695cad042677a588480c86e6')
+    version(
+        '0.10', sha256='38a4d6e242b8bab693cd83a5f5ade3d816463b498658e7ab14ce64c4d458c88b')
+    version(
+        '0.9', sha256='32d8a9a9d63f4f81194c0014b3b742679dce81a26d45127d9810a68a561fe4e2')
+    version(
+        '0.8.1', sha256='afcf31443a478c32bbac4b00337ee9026a13d0e2ac83d30c79151462513bb0d4')
```

Comment:
These can be on a single line, we exempt checksums from line length requirements.

Reply:
Placed them back on the same line as version